flowgger.toml

```diff
@@ -160,3 +160,11 @@ framing = "line"
 format = "rfc3164"
 # Format of the optional timestamp to be prepended to each event
 syslog_prepend_timestamp="[[[year]-[month]-[day]T[hour]:[minute]:[second].[subsecond digits:6]Z]"
+
+
+[test]
+
+###################
+#  Fuzzing Test   #
+###################
+fuzzed_message_count = 500
```

Comment:
why is this in the prod config ? this should be in the test file as constant or in a dedicated file. not in the config file

Reply:
will do. Thanks